tests/recovery/recovery_test/main_test.go

```diff
@@ -13,6 +13,8 @@ import (
 
 const (
 	dataSubPath = "recovery-data"
+	dirPath     = "kopia_dummy_repo"
```

Comment:
Can you use a more descriptive name? Something like consistencyTestRepo or similar? Same thing goes for the path `kopia_dummy_repo`.

Reply:
Please address this. The variable name `dirPath` does not describe its purpose. How about crashConsistencyDirPath or whatever the purpose of this variable is.
Similarly, `kopia_dummy_repo` does not describe its purpose. How about something like - crash-consistency-test-repo